logic/src/commonMain/kotlin/com/wire/kalium/logic/data/e2ei/AcmeAuthorization.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Wire
+ * Copyright (C) 2024 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ */
+package com.wire.kalium.logic.data.e2ei
+
+import com.wire.kalium.cryptography.NewAcmeAuthz
+
+data class AcmeAuthorization(
+    val nonce: String,
+    val location: String?,
+    val response: ByteArray,
+    val challengeType: AuthorizationChallengeType,
+    val newAcmeAuthz: NewAcmeAuthz
+)
+
+enum class AuthorizationChallengeType {
+    DPoP, OIDC
```

Comment:
Just to be sure we can search by its meaning as well in case there are other abbreviations that could be confused when Googling it.
```suggestion
    /**
     * Data Protection on Demand
     */
    DPoP, 
    /**
     * OpenID Connect
     */
    OIDC
```